src/content/structured/get-started/custom-theme.mdx

```diff
@@ -39,9 +44,144 @@ Alternatively, you can include the `ic-theme` component and set the `color` prop
 <ic-theme color="#5c0949"></ic-theme>
```

Comment:
Awaiting this work to implement the website changes for this: https://github.com/mi6/ic-ui-kit/issues/2318

Reply:
Sounds like @ad9242 is aiming to get this in after demo, so might need a follow up ticket to be made (and let me know the number) to do this as soon as demo is over, with it not being done here